### PR TITLE
extract chapters for policy overview, apply, testing, policies

### PR DESCRIPTION
**Pull request by JimBugwadia**
The intent is to make it easier to for new users to find relevant content (some users had a hard time getting started.)

Signed-off-by: Jim Bugwadia <jim@nirmata.com>

### CODE REVIEW

**Comment by JimBugwadia** [comment]
[Preview](https://deploy-preview-133--kyverno.netlify.app/docs/)

**Review by realshuting — APPROVED** [review]

File: content/en/docs/Applying Policies/_index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Applying Policies
+description: Apply policies across clusters and your continious delivery pipelines
```
Comment (chipzoller) [inline content/en/docs/Applying Policies/_index.md:3]: "...your continuous delivery..."

File: content/en/docs/Applying Policies/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## In CI/CD Pipelines
+
+You can use the [Kyverno CLI](http://localhost:1313/docs/kyverno-cli/#apply) to apply policies to YAML resource manifest files as part of your continious delivery pipeline. This command line tool allows integrating Kyverno into a GitOps style workflows and checking for policy compliance of resource manifests before they are committed.
```
Comment (chipzoller) [inline content/en/docs/Applying Policies/_index.md:25]: "...as part of your continuous delivery..."

File: content/en/docs/Applying Policies/_index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+{{% alert title="Tip" color="info" %}}
+The [Kyverno Policies](/policies/) repository contains several policies you can immediately apply to your clusters.
+{{% /alert %}}
+
+
+## In Cluster
+
+On installation, Kyverno runs as a [dynamic admission controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) in a Kubernetes cluster. Kyverno receives validating and mutating admission webhook HTTP callbacks from the kube-apiserver and applies matching policies to return results that enforce admission policies or reject requests.
+
+Policies with validation rules can be used to block insecure or non-compliant configurations by setting the [`validationFailureAction`](http://localhost:1313/docs/writing-policies/validate/#validation-failure-action) to `enforce`. Or, validation rules can be applied using periodic scans with results available as [policy reports](/docs/policy-reports/).
```
Comment (chipzoller) [inline content/en/docs/Applying Policies/_index.md:16]: Change link to relative link.

File: content/en/docs/Applying Policies/_index.md (hunk 2 above)
Comment (chipzoller) [inline content/en/docs/Applying Policies/_index.md:25]: "This command line tool allows integrating Kyverno into a GitOps style workflow and checks for policy compliance of resource manifests before they are committed."

File: content/en/docs/Applying Policies/_index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## In CI/CD Pipelines
+
+You can use the [Kyverno CLI](http://localhost:1313/docs/kyverno-cli/#apply) to apply policies to YAML resource manifest files as part of your continious delivery pipeline. This command line tool allows integrating Kyverno into a GitOps style workflows and checking for policy compliance of resource manifests before they are committed.
+
```
Comment (chipzoller) [inline content/en/docs/Applying Policies/_index.md:26]: New lines can be eliminated.

File: content/en/docs/CRDs/_index.md [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
+
+The complete Kyverno CRD reference can be viewed using this link:
+
+> **[kyverno.io/v1.ClusterPolicy](https://htmlpreview.github.io/?https://github.com/kyverno/kyverno/blob/main/docs/crd/v1/index.html#kyverno.io/v1.ClusterPolicy)**
+
+The HTML source is available in the [Kyverno GitHub repository](https://github.com/kyverno/kyverno/tree/main/docs) and generated from type definitions stored at [kyverno/kyverno/pkg/api](https://github.com/kyverno/kyverno/tree/main/pkg/api).
+
+## kubectl explain 
+
+When operating in a Kubernetes cluster with Kyverno installed, you can always inspect Kyverno types natively using `kubectl explain`.
```
Comment (chipzoller) [inline content/en/docs/CRDs/_index.md:18]: Does it make sense to write a small note about how this also enables IDE integrations, ex., VS Code? If not here, maybe elsewhere? It's a very nice feature and many may not know the two pieces of resulting functionality come from the same thing.

File: content/en/docs/Kyverno Policies/_index.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+title: Kyverno Policies
+description: Learn how Kyverno policies and rules work.
+weight: 45
+---
+
+A Kyverno policy is a collection of rules. Each rule consists of a [`match`](/docs/writing-policies/match-exclude/) clause, an optional [`exclude`](/docs/writing-policies/match-exclude/) clause, and one of a [`validate`](/docs/writing-policies/validate/), [`mutate`](/docs/writing-policies/mutate/), or [`generate`](/docs/writing-policies/generate) clause. A rule definition can contain only a single `validate`, `mutate`, or `generate` child node.
+
+<img src="/images/Kyverno-Policy-Structure.png" alt="Kyverno Policy" width="50%"/>
```
Comment (chipzoller) [inline content/en/docs/Kyverno Policies/_index.md:9]: Scaling this image down was a good idea. I too thought it was quite large.

File: content/en/docs/Testing Policies/_index.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Testing Policies"
+linkTitle: "Testing Policies"
+weight: 55
+description: >
+    Manage success and failure test scenarios for policies.
+---
+
+The [Kyverno CLI `test` command](http://localhost:1313/docs/kyverno-cli/#test) can be used to test if a policy is behaving as expected.
```
Comment (chipzoller) [inline content/en/docs/Testing Policies/_index.md:9]: Relative link only.

File: content/en/docs/Testing Policies/_index.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+Please refer to the [`test` command documentation](http://localhost:1313/docs/kyverno-cli/#test) for details. You can also find examples in the [sample policies repository](https://github.com/kyverno/policies/tree/main/pod-security/tests).
```
Comment (chipzoller) [inline content/en/docs/Testing Policies/_index.md:11]: First link to be relative.

**Review by chipzoller — COMMENTED** [review]
New structure looks good. Only a few minor things.

File: content/en/docs/CRDs/_index.md (hunk 5 above)
Comment (JimBugwadia) [reply content/en/docs/CRDs/_index.md:18]: Yes, I agree. Perhaps we can pull the `kubectl explain` and VS Code integration sections into Writing policies or a new `Validating Policies` section? We also have community contributions for GitHub precommit actions to help validate policies, we can also reference that.

Comment (JimBugwadia) [reply content/en/docs/CRDs/_index.md:18]: I will log a separate issue to track that.

Comment (JimBugwadia) [reply content/en/docs/CRDs/_index.md:18]: Logged: https://github.com/kyverno/website/issues/135

**Pull request merged**